impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -199,61 +248,87 @@ SharedHandleVK<vk::RenderPass> RenderPassVK::CreateVKRenderPass(
 
 RenderPassVK::RenderPassVK(const std::shared_ptr<const Context>& context,
```

Comment:
Note that the pass bindings cache is mostly removed, as I was not able to measure a performance improvement from it.